### PR TITLE
Nest Migration

### PR DESCRIPTION
**Pull request by willstenzel**
It's a chonker of a PR but here it is 😄

### CODE REVIEW

File: packages/server/.env.development [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PUBLICKEY=BICM9ljt6UzPC2HsKwlGotdrOSjva59TcwEnolgjf67P8BLN-UStYY2kvTyKDT5DpqJX0sK1z5ZyCdc98_z-AjY
 PRIVATEKEY=3134tddi2nyoxnamvbImuban4fXXgiiXia4t3SGVOBA
 EMAIL=mailto:zuck@fb.com
-TWILIOACCOUNTSID=REMEMBERTHEALAMO
+TWILIOACCOUNTSID=ACREMEMBERTHEALAMO
 TWILIOAUTHTOKEN=FLYMETOTHEMOONANDLETMESINGFOREVERMORE
 TWILIOPHONENUMBER=+14204206969
-COOKIE_PASSWORD=chunguschunguschunguschunguschunguschunguschunguschunguschungus 
\ No newline at end of file
+JWT_SECRET=chunguschunguschunguschunguschunguschunguschunguschunguschungus
```
Comment (NEUDitao) [inline packages/server/.env.development:7]: what's the `JWT_SECRET`?

Comment (willstenzel) [reply packages/server/.env.development:7]: `JWT_SECRET` is the private key used to secure the auth cookie which is set in the user browser

File: packages/server/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<p align="center">
```
Comment (NEUDitao) [inline packages/server/README.md:1]: can we get rid of this?

File: packages/server/src/notification/desktop-notif-subscriber.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    this.notifService = notifService;
+    connection.subscribers.push(this);
+  }
+
+  listenTo() {
+    return DesktopNotifModel;
+  }
+
+  async afterInsert(event: InsertEvent<DesktopNotifModel>) {
+    console.log(`BEFORE DESKTOPNOTIF INSERTED: `, event.entity);
```
Comment (liustanley) [inline packages/server/src/notification/desktop-notif-subscriber.ts:24]: Not sure if this console.log is meant for debugging or logging, if it's not logging let's remove it

File: packages/server/src/queue/queue.entity.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+  @Column('text')
+  room: string;
+
+  @OneToMany((type) => QuestionModel, (qm) => qm.queue)
+  @Exclude()
+  questions: QuestionModel[];
+
+  @Expose()
+  get staffList(): UserPartial[] {
```
Comment (NEUDitao) [inline packages/server/src/queue/queue.entity.ts:36]: nit: can we just leave a TODO here so we know that this is temp wh ile we figure out OH -> queues?

**Pull request merged**